### PR TITLE
fix validation error for exemption list#174 

### PR DESCRIPTION
**Pull request by sgrg1**
fix showing correct validation error on exemption certificate

### CODE REVIEW

File: src/common/ExemptionUtilities.js [hunk 1]
```diff
@@ -2,22 +2,28 @@
  * Quick Solution for Validating Exemption Errors Outside of Formik
  * @param {object} exemption object representation of an exemption
  */
-const GetExemptionFormErrors = exemption => {
+const GetExemptionFormErrors = (exemption, exemptionErrorCheck=true) => {
```
Comment (martypowell) [inline src/common/ExemptionUtilities.js]: looks like your formatting is off here.

Comment (sgrg1) [reply src/common/ExemptionUtilities.js]: fixed

Comment (martypowell) [reply src/common/ExemptionUtilities.js]: I don't understand what `exemptionErrorCheck` means.

Comment (sgrg1) [reply src/common/ExemptionUtilities.js]: Please give me the name of variable what ever make sense to you

Comment (martypowell) [reply src/common/ExemptionUtilities.js]: per our chat, i don't understand the code so I can't give you a name for the variable, but i'm not sure this change will persist after this is resolved. So i wouldn't get too hung up on this.

**Comment by martypowell** [comment]
ok @sgrg1 this first step to this will be to move the [ExemptionList](https://github.com/baltimorecounty/react-transient-tax/blob/integration/src/components/ExemptionCertificateField.jsx#L58) out of the `ExemptionCertificateField` and into the `ExemptionCertificateForm`.

I would expect that to be put here.

image

This will allow us to remove the complexity of having the list and the form together, and make it easier to fix the validation the formik way.

This should be submitted as a pr independent of this one.

If you need any help doing this, or have questions, please don't hesitate to ask.

**Comment by martypowell** [comment]
closing in favor of #187

**Pull request closed without merging**